default.json

```diff
@@ -3,6 +3,11 @@
   "channelName": "general",
   "channelId": "channelIdHere",
 
+  "officeHours": {
```

Comment:
I'd add a `on` Boolean property that defaults to "off", set the default `begin`/`end` values to 9, 17 respectively (since that is the understood 9-5 work hours). 

It'd be an easy additional check to see if the bot should respect the designated office hours or not. Otherwise it looks good.